.github/actions/git-diff-on-components/src/index.js

```diff
@@ -336,20 +344,35 @@ async function run() {
     componentsDiffContents = await checkVersionModification(componentsPendingForGitDiff, componentsThatDidNotModifyVersion);
   }
 
-  const totalErrors = componentsThatDidNotModifyVersion.length + componentsDiffContents.length;
+  const totalErrors = componentsThatDidNotModifyVersion.length;
   let counter = 1;
 
+  await execCmd("git", ["config", "--global", "user.email", "lucascarezia@gmail.com"]);
+  await execCmd("git", ["config", "--global", "user.name", "Lucas Caresia"]);
```

Comment:
Are these lines needed for the `git commit`?
If so, we better use Pipedream data, like
```
email: pipedream@pipedream.com
name: Pipedream
```

Reply:
Yes, unfortunately is necessary.